crates/modelardb_server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -297,24 +380,32 @@ impl PhysicalExpr for ModelCountPhysicalExpr {
         Ok(ColumnarValue::Array(Arc::new(result.finish())))
     }
 
+    /// Return the list of [`PhysicalExprs`](PhysicalExpr) that provides input to this [`PhysicalExpr`].
```

Comment:
provide ?

Reply:
Fixed for all instances for `children()`.